nautobot/docs/development/getting-started.md

```diff
@@ -648,6 +648,25 @@ To enforce best practices around consistent [coding style](style-guide.md), Naut
 | `invoke flake8`         | `flake8`                     |
 | `invoke black`          | `black`                      |
 
+## Working on Documentation
+
+Some features require documentation updates or new documentation to be written. The documentation files can be found in the `docs` directory. To preview these changes locally, you can use `mkdocs`.
+
+### Installing `mkdocs`
+
+The `mkdocs` command can be installed via pip, either globally or in your virtual environment.
+
+```
+pip install mkdocs mkdocs mkdocs-include-markdown-plugin
+```
```

Comment:
Probably worth noting that if you are using `poetry`, `poetry install` will automatically install the mkdocs dependencies and make it available in the `poetry shell` virtualenv - in which case there is no need to do this step separately.

Reply:
Done, let me know whether the note works for you!